README.md

```diff
@@ -32,6 +32,11 @@ The CentrePoint API uses [OAuth 2.0](https://oauth.net/2/) for its authorization
 * Retrieval of listing of webhook events. (see [Webhook Events](./sections/webhook_events.md))
 * Retrieval of a list of the webhook subscriptions for a particular study as well as the history or webhook requests. (see [Webhooks](./sections/webhooks.md))
 
+#### CentrePoint Designer
+
+* Retrieval of completed and not depreciated Study Programs. (see [Study Programs](./sections/study_program.md))
```

Comment:
should be "deprecated"

Reply:
I had removed that section for CentrePoint Designer. Moved the reference line referring to the Study Programs documentation.